src/SelectSingle.vue

```diff
@@ -249,7 +263,7 @@
 					'option-current': index == activeIndex
 				}"
 				role="option"
-				:aria-selected="index == activeIndex"
+				:aria-selected="index == selectedIndex ? 'true' : 'false'"
```

Comment:
thank you 🙇

Reply:
im not sure about that ^^
aria-selected is set as `true` when we navigate using keyboard. In this case `activeIndex` changes not selectedIndex.
@jackkoppa  @RED143

Reply:
@ee923925github we can't use activeIndex here, because it is a logically wrong. If I get it right aria-selected property means option which is selected in select. If we will be use activeIndex all options will be selected for keyboard user. Behaviour that you described is similar to aria-current property.

@jackkoppa

Reply:
Thanks @ivanshtyrlyaev & @RED143!

I feel like I'm very bad at this because it took me a long time to work through both the PR & this particular distinction. In the end, I believe this PR is the correct approach, because `aria-selected` is used to determine whether the options in the list should be read as the *currently selected* option while someone is keying through those options.

Thanks to the PR deploy, we can see this in action. Current Storybook, as @RED143 describes, is not reading any of the options as "not selected", because when they use `activeIndex`, they're all read as `aria-selected`:
![Screen Recording 2020-12-11 at 02 10 23 AM](https://user-images.githubusercontent.com/14101491/101874553-18406900-3b57-11eb-9e3b-77fe789f4583.gif)

After this PR, though, every option that's not the *currently selected* option gets read out, correctly, as "Not Selected":
![Screen Recording 2020-12-11 at 02 09 44 AM](https://user-images.githubusercontent.com/14101491/101874598-28f0df00-3b57-11eb-9f46-4cf4d7e0dba0.gif)


So let's go through with this change! Thanks for the discussion, both of you - appreciate the help in figuring out what's going on here 😬